prometheus/README.md

```diff
@@ -32,60 +32,41 @@ please see Install Grafana
 ```
 
 
-## Install the Prometheus Operator
+## Install Prometheus and the Prometheus Operator
```

Comment:
Can we get an initial paragraph explaining that this is the StorageOS Prometheus use case, and something like 'for more information, please see our Prometheus use case documentation at {link} or similar? The intent of this repo is to be an adjunct to the docs, not fully independent.